src/model/repos.rs

```diff
@@ -215,3 +215,53 @@ pub struct ExternalWiki {
     /// URL of external wiki.
     pub external_wiki_url: String,
 }
+
+/// Links contains related resource URLs for a file entry.
+#[derive(Default, Debug, Clone, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Links {
```

Comment:
```suggestion
pub struct FileLinks {
```

I feel like `Links` is a bit too non-descriptive, maybe `FileLinks` is a better name

Reply:
Thanks for pointing it out. Indeed it does seem like a more meaningful name, changes has been applied on #19 & #20 :+1: